### ISSUE TITLE
new install open tutorial.srctrlprj failed

### ISSUE THREAD

**Issue by wenmin92** [CoatiSoftware/SourcetrailBugTracker#733]
I use PORTABLE version 2019.3.46 (64bit), new install, then open tutorial.srctrlprj failed.
<details>
<summary>log_2019-09-14_08-39-05.txt</summary>

```log
08:39:05 | 8040 | LogManager.cpp:41 LogManager::setLoggingEnabled() | INFO: Enabled logging for Sourcetrail 64 bit, version 2019.3.46
08:39:05 | 8040 | SharedMemoryGarbageCollector.cpp:58 SharedMemoryGarbageCollector::run() | INFO: start shared memory garbage collection
08:39:10 | 16252 | INFO: send MessageStatus Starting Sourcetrail 64 bit, version 2019.3.46
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Fira Sans
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Fira Sans
08:39:10 | 16252 | StatusBarController.cpp:89 StatusBarController::setStatus() | INFO: STATUS Starting Sourcetrail 64 bit, version 2019.3.46
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Source Code Pro
08:39:10 | 16252 | INFO: send MessageStatus Load settings: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/ApplicationSettings.xml
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Source Code Pro
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Source Code Pro
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Roboto
08:39:10 | 8040 | utilityQt.cpp:66 utility::loadFontsFromDirectory() | INFO: Loaded FontFamily: Roboto
08:39:10 | 16252 | StatusBarController.cpp:89 StatusBarController::setStatus() | INFO: STATUS Load settings: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/ApplicationSettings.xml
08:39:10 | 16252 | INFO: send MessageStatus Enabled console and file logging.
08:39:10 | 16252 | StatusBarController.cpp:89 StatusBarController::setStatus() | INFO: STATUS Enabled console and file logging.
08:39:10 | 16252 | INFO: send MessagePingReceived 
08:39:10 | 16252 | INFO: send MessageLoadProject , settingsChanged: false, refreshMode: 0
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessageWindowFocus 
08:39:10 | 16252 | INFO: send MessagePingReceived 
08:39:14 | 16252 | INFO: send MessageWindowFocus 
08:39:14 | 16252 | INFO: send MessageLoadProject C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/./projects/tutorial/tutorial.srctrlprj, settingsChanged: false, refreshMode: 0
08:39:14 | 16252 | INFO: send MessagePingReceived 
08:39:18 | 16252 | INFO: send MessagePingReceived 
08:39:23 | 16252 | INFO: send MessagePingReceived 
08:39:27 | 16252 | INFO: send MessageStatus Loading Project: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/./projects/tutorial/tutorial.srctrlprj - loading
08:39:27 | 16252 | StatusBarController.cpp:89 StatusBarController::setStatus() | INFO: STATUS Loading Project: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/./projects/tutorial/tutorial.srctrlprj
08:39:27 | 7416 | Application.cpp:298 Application::handleMessage() | ERROR: Failed to load project, unknown exception thrown.
08:39:27 | 16252 | INFO: send MessageStatus Failed to load project, unknown exception was thrown: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/./projects/tutorial/tutorial.srctrlprj - error
08:39:27 | 16252 | StatusBarController.cpp:89 StatusBarController::setStatus() | INFO: STATUS Failed to load project, unknown exception was thrown: C:/Users/wenmin92/scoop/apps/sourcetrail/current/user/./projects/tutorial/tutorial.srctrlprj
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\14.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VCExpress\14.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VCExpress\14.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\12.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\12.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VCExpress\12.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VCExpress\12.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\11.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\11.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VCExpress\11.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VCExpress\11.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\10.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\10.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VCExpress\10.0
08:39:28 | 8040 | CxxVs10To14HeaderPathDetector.cpp:97 CxxVs10To14HeaderPathDetector::getVsInstallPathUsingRegistry() | INFO: Found working regestry key for VS install path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VCExpress\10.0
08:40:01 | 16252 | INFO: send MessageWindowFocus 
```

</details>

**Comment by mlangkabel** [comment]
Hi @wenmin92, thanks for your report. Am I correct with the assumption that you extracted the portable archive to `C:/Users/wenmin92/scoop/apps/sourcetrail/current`? Maybe Sourcetrail has issues to get write access at that path. Can you try to put it to a different path?

**Comment by wenmin92** [comment]
Hi @mlangkabel , I have known what happened. the `current` folder is a directory junction, see below:

![image](https://user-images.githubusercontent.com/6901815/64960149-476d1980-d8c5-11e9-81e9-5412de0f0352.png)

When open source trail from the origin `2019.3.46` folder, that's OK.

**Comment by mlangkabel** [comment]
Thanks for the update, we will look into it :)

**Comment by mlangkabel** [comment]
We managed to reproduce the issue.

**Comment by mlangkabel** [comment]
Fixed with Sourcetrail 2019.4.28
